### PR TITLE
[WNMGDS-2236] Initial redesign of hgov footer

### PR DESCRIPTION
**Pull request by zarahzachz**
WNMGDS-2236

Update the hgov footer to match Brent's design.

[Demo url](https://cmsgov.github.io/design-system/branch/WNMGDS-2236/update-hgov-footer/storybook/?path=/story/healthcare-footer--basic-footer&globals=theme:healthcare)

### CODE REVIEW

**Review by brentschneider — CHANGES_REQUESTED** [review]
Overall spot on from a visual perspective,   
found the url on the HC.gov logo points to HHS.gov  
other than the url 👏🏼👏🏼👏🏼👏🏼 yay!

**Review by pwolfert — COMMENTED** [review]
Looking at these screenshots, there appears to be a difference in how the healthcare.gov logo aligns between Firefox and Webkit. Webkit seems wrong

**Comment by forrestbaer** [comment]
Will need Labels/etc also please!

File: packages/ds-healthcare-gov/src/styles/components/_Logo.scss [hunk 1]
```diff
@@ -1,13 +1,3 @@
-.hc-c-footer__logo {
-  max-width: 100%;
-  vertical-align: top;
-
-  svg {
-    fill: var(--color-muted);
-    width: 100%;
-  }
-}
```
Comment (pwolfert) [inline packages/ds-healthcare-gov/src/styles/components/_Logo.scss:9]: Nice, I love it when a fix involves deleting code 😄

**Pull request merged**